packages/react/src/components/Dialog/Dialog.tsx

```diff
@@ -88,6 +88,8 @@ export interface DialogProps
 }
 
 export interface DialogState {
+  contentId?: string
```

Comment:
Shouldn't we use these as an id of the elements, or they are used only for the aria-* attributes?

Reply:
I decided to keep responsibility for this on accessibility behaviors: https://github.com/stardust-ui/react/pull/1449/files#diff-7d8c4d117a2402a4e3fbb997e3c6693fL32